rollup/Makefile

```diff
@@ -7,9 +7,22 @@ mock_abi:
 	cd .. && solc --evm-version cancun --bin --abi --optimize --overwrite -o ./build/bin ./rollup/mock_bridge/MockBridge.sol
 	cd .. && go run github.com/scroll-tech/go-ethereum/cmd/abigen --bin=./build/bin/MockBridge.bin --abi=./build/bin/MockBridge.abi --pkg=mock_bridge --out=./rollup/mock_bridge/MockBridge.go
 
-rollup_bins: ## Builds the Rollup bins.
+libzstd:
+	sudo mkdir -p /scroll/lib/
```

Comment:
Is this meant to be run in the `dev_docker` container? And why do we need `sudo`?

Reply:
because in CI, it needs sudo to create a new dir from root dir.